src/CompletionHandler.php

```diff
@@ -442,4 +435,19 @@ protected function getAllOptions()
             $this->application->getDefinition()->getOptions()
         );
     }
+
+    protected function getCommandNames()
+    {
+        $commands = array();
+
+        foreach ($this->application->all() as $command) {
+            if ($command->isHidden()) {
+                continue;
+            }
+
+            $commands[] = $command->getName();
```

Comment:
Use `$name` instead of `$command->getName()`.

---

Optimization, because we reduce amount of made function calls. Probably would see benefit only for applications with large command sets. Anyway this is how it worked before this PR.